tests/functional/spec/thank_you.spec.js

```diff
@@ -39,3 +41,22 @@ describe("Thank You Default", () => {
     });
   });
 });
+
+describe("Thank You Default View Response Enabled", () => {
+  describe("Given I launch a questionnaire where view response is enabled", () => {
+    beforeEach(() => {
+      browser.openQuestionnaire("test_thank_you.json");
```

Comment:
Is there a reason for adding this to the thank you page schema and tests, rather than the view submitted response schema and tests?

Reply:
I added it here as I wanted to test that the `view-answers` link etc is available on the thank you page, I can move it if you think it's better with the view submitted response tests?

Reply:
Ah ok, I think that does make sense in this test, but raises questions about what the view submitted response schema is for. Given we have a card to review and rationalise the schemas, I think this is fine for now.